charts/spire/README.md

```diff
@@ -15,7 +57,6 @@ A Helm chart for deploying the complete Spire stack including: spire-server, spi
 
 | Dependency | Supported Versions |
 |:-----------|:-------------------|
-| SPIRE      | `1.5.3+`, `1.6.3+` |
```

Comment:
Only support the version of spire the chart is tested/released with, so no reason to explicitly mention a range.